features/product/viewing_products/viewing_product_sorted_by_date.feature

```diff
@@ -0,0 +1,27 @@
+@viewing_products
+Feature: Sorting listed products
+    In order to change the order by which products are displayed
+    As an Administrator
+    I want to sort products
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Berserk Pug" with code "B_PUG"
+        And this product was created at "05-10-2016"
```

Comment:
Maybe we could merge this two steps into ``And the store has a product "Berserk Pug" with code "B_PUG", created at "05-10-2016"``? I would significantly reduce background size.